### PR TITLE
Bug 1777129: Tighten cipher usage to library-go

### PR DESCRIPTION
**Pull request by jhadvig**
Use the cipher suites that are provided by the OpenShift library-go as the reasonable defaults, based on https://github.com/openshift/oauth-proxy/pull/130

/assign @spadgett 
cc'ing @stlaz

### CODE REVIEW

File: cmd/bridge/main.go [hunk 1]
```diff
@@ -19,12 +19,12 @@ import (
 
 	"github.com/openshift/console/pkg/auth"
 	"github.com/openshift/console/pkg/bridge"
-	"github.com/openshift/console/pkg/crypto"
```
Comment (spadgett) [inline cmd/bridge/main.go:22]: Can you remove `pkg/crypto` from console if it's no longer needed?

Comment (jhadvig) [reply cmd/bridge/main.go:22]: yeah, we probably should!

Comment (jhadvig) [reply cmd/bridge/main.go:22]: The line you you are mentioning was changed but the second commit:
Screenshot 2020-08-11 at 15 44 31

**Comment by jhadvig** [comment]
@spadgett I've removed the console's `crypto` pkg.
PTAL :)

**Pull request merged**